pages/nft/iscn/_iscnId.vue

```diff
@@ -336,7 +340,8 @@ export default class NFTTestMintPage extends Vue {
   }
 
   get NftName() {
-    return `${this.isWritingNFT ? 'Writing NFT - ' : ''}${this.iscnData?.contentMetadata?.name || 'NFT'}`;
+    const prefix = this.NFTPrefix || `${this.isWritingNFT ? 'Writing NFT' : ''}`;
```

Comment:
Seem `this.NFTPrefix` is used in this case??

Reply:
Lets handle the isWritingNFT judgement in `this.NFTPrefix`